openmdao/visualization/n2_viewer/n2_viewer.py

```diff
@@ -116,33 +148,59 @@ def _get_tree_dict(system, component_execution_orders, component_execution_index
     if isinstance(system, ImplicitComponent):
         if overrides_method('solve_linear', system, ImplicitComponent):
             tree_dict['linear_solver'] = "solve_linear"
+            tree_dict['linear_solver_options'] = None
         elif system.linear_solver:
             tree_dict['linear_solver'] = system.linear_solver.SOLVER
+            options = {k: system.linear_solver.options[k] for k in system.linear_solver.options}
+            tree_dict['linear_solver_options'] = options
         else:
             tree_dict['linear_solver'] = ""
+            tree_dict['linear_solver_options'] = None
 
         if overrides_method('solve_nonlinear', system, ImplicitComponent):
             tree_dict['nonlinear_solver'] = "solve_nonlinear"
+            tree_dict['nonlinear_solver_options'] = None
         elif system.nonlinear_solver:
             tree_dict['nonlinear_solver'] = system.nonlinear_solver.SOLVER
+            options = {k: system.nonlinear_solver.options[k]
+                       for k in system.nonlinear_solver.options}
+            tree_dict['nonlinear_solver_options'] = options
         else:
             tree_dict['nonlinear_solver'] = ""
+            tree_dict['nonlinear_solver_options'] = None
     else:
         if system.linear_solver:
             tree_dict['linear_solver'] = system.linear_solver.SOLVER
+            options = {k: system.linear_solver.options[k] for k in system.linear_solver.options}
+            tree_dict['linear_solver_options'] = options
         else:
             tree_dict['linear_solver'] = ""
+            tree_dict['linear_solver_options'] = None
 
         if system.nonlinear_solver:
             tree_dict['nonlinear_solver'] = system.nonlinear_solver.SOLVER
+            options = {k: system.nonlinear_solver.options[k]
+                       for k in system.nonlinear_solver.options}
+            tree_dict['nonlinear_solver_options'] = options
 
             if system.nonlinear_solver.SOLVER == NewtonSolver.SOLVER:
                 tree_dict['solve_subsystems'] = system._nonlinear_solver.options['solve_subsystems']
         else:
             tree_dict['nonlinear_solver'] = ""
+            tree_dict['nonlinear_solver_options'] = None
 
     tree_dict['children'] = children
 
+    options = {}
+    from openmdao.solvers.solver import Solver
```

Comment:
Is this import used?

Reply:
no. left over from some code that used to be there. Will remove. Thanks